src/main/java/at/hannibal2/skyhanni/features/misc/GeorgeDisplay.kt

```diff
@@ -0,0 +1,86 @@
+    )
+
+    private val display = mutableListOf<Renderable>()
+
+    @SubscribeEvent
+    fun onInventoryOpen(event: InventoryOpenEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+        if (!isEnabled()) return
+        if (event.inventoryName != "Offer Pets") return
+        val stack = event.inventoryItems[ROW_OFFSET + INDEX_OFFSET] ?: return
+        if (stack.cleanName() != "+1 Taming Level Cap") return
+        val updateList: MutableList<Renderable> = mutableListOf<Renderable>(Renderable.string("§d§lTaming 60 Cost:"))
+        var totalCost: Double = 0.0
+        loop@ for (line in stack.getLore()) {
+            neededPetPattern.matchMatcher(line) {
+                val origTierString = group("tier")
+                val tier = when (origTierString.uppercase()) {
+                    "COMMON" -> "0"
+                    "UNCOMMON" -> "1"
+                    "RARE" -> "2"
+                    "EPIC" -> "3"
+                    "LEGENDARY" -> "4"
+                    "MYTHIC" -> "5"
+                    else -> ""
+                }
+                val origPetString = group("pet") ?: ""
+                val pet = origPetString.uppercase().replace(" ", "_").removePrefix("FROST_")
+                val petPrice = "$pet;$tier".asInternalName().getPriceOrNull() ?: continue@loop
+                totalCost += petPrice
+                updateList.add(Renderable.clickAndHover(
+                    text = " §7- ${group("fullThing")}§7: §6${petPrice.addSeparators()} coins",
+                    tips = listOf("§aClick to run §e/ahsearch [Lvl 1] $origPetString §ato find this pet on the Auction House.", "§aNotes: §eMake sure to set the rarity filter to ${group("tierColorCodes")}$origTierString §eon your own! §cBooster Cookie required!"),
+                    onClick = {
+                        LorenzUtils.sendCommandToServer("ahsearch [Lvl 1] $origPetString")
+                    }
+                ))
+            }
+        }
+        updateList.addAll(listOf<Renderable>(Renderable.string("§dTotal cost §7(§6Lowest BIN§7): §6${totalCost.addSeparators()} coins"), Renderable.string("§c§lDisclaimer:§r§c Some pets are not available on the Auction House.")))
+        updateList.update()
+    }
+
+    private fun List<Renderable>.update() {
+        display.clear()
+        display.addAll(this)
+    }
+
+    @SubscribeEvent
+    fun onRenderOverlay(event: GuiRenderEvent.ChestGuiOverlayRenderEvent) {
+        if (!LorenzUtils.inSkyBlock) return
+        if (!isEnabled()) return
+        if (InventoryUtils.openInventoryName() != "Offer Pets") return
+        config.pos.renderRenderables(display, posLabel = "Taming 60 Progress")
```

Comment:
Please use the same GUI label name as the config toggle name

Reply:
done